docs/testing/debug-office-add-ins-on-ipad-and-mac.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Debug Office Add-ins on iPad and Mac
+title: Debug Office Add-ins on Mac
 description: ''
-ms.date: 03/21/2018
+ms.date: 04/24/2019
 localization_priority: Priority
 ---
 
-# Debug Office Add-ins on iPad and Mac
+# Debug Office Add-ins on Mac
 
-You can use Visual Studio to develop and debug add-ins on Windows, but you can't use it to debug add-ins on the iPad or Mac. Because add-ins are developed using HTML and Javascript, they are designed to work across platforms, but there might be subtle differences in how different browsers render the HTML. This article describes how to debug add-ins running on an iPad or Mac. 
+You can use Visual Studio to develop and debug add-ins on Windows, but you can't use it to debug add-ins on the Mac. Because add-ins are developed using HTML and Javascript, they are designed to work across platforms, but there might be subtle differences in how different browsers render the HTML. This article describes how to debug add-ins running on a Mac.
```

Comment:
```suggestion
You can use Visual Studio to develop and debug add-ins on Windows, but you can't use it to debug add-ins on a Mac. Because add-ins are developed using HTML and JavaScript, they are designed to work across platforms, but there might be subtle differences in how different browsers render the HTML. This article describes how to debug add-ins running on a Mac.
```